components/04-views/dashboard/account-info/account-info.js

```diff
@@ -0,0 +1,66 @@
+'use strict'
```

Comment:
This file should be moved and bind to `dashboard-form` module

Reply:
But this script exists only on this issue in `account-info` tab. I remember about reusable components but it's really specific script. And this logic from file will be not merged into magento. I hope we have in magento something other than my file. It's only for preview.

Reply:
I think it should be move to module and use it in view. If it's only for preview, please add some comment in readme or inside js file. In Magento there is a script for this but...  what if we would like to remove magento js logic... we won't have any script for this.

Reply:
moved